### PR TITLE
Implemented all Security/Random Cast tests

### PR DESCRIPTION
**Pull request by ruudboon**
Implemented all Security/Random Cast tests. Resolves refactoring RandomCest.php. See #13655

### CODE REVIEW

File: tests/unit/Security/Random/Base64SafeCest.php [hunk 1]
```diff
@@ -32,6 +32,36 @@ class Base64SafeCest
...
+            [12, true, 'a-z0-9_=-'],
+            [16, false, 'a-z0-9_-'],
+            [16, true, 'a-z0-9_=-'],
+            [24, false, 'a-z0-9_-'],
+            [24, true, 'a-z0-9_=-'],
+            [48, false, 'a-z0-9_-'],
+            [48, true, 'a-z0-9_=-'],
+            [100, false, 'a-z0-9_-'],
+            [100, true, 'a-z0-9_=-'],
+        ];
```
Comment (niden) [inline tests/unit/Security/Random/Base64SafeCest.php]: You can also use the `Codeception\Example` class to inject these examples in your test automatically. Have a look at the `Text::camelize` tests

**Review by niden — DISMISSED** [review]

**Comment by niden** [comment]
This needs `phpcbf`

**Comment by niden** [comment]
Also if you can rebase that will remove the interface errors the build had. Thanks

**Comment by ruudboon** [comment]
No prob. I think I forgot to commit something. Will rebase and push the changes soon.

File: tests/unit/Security/Random/Base64SafeCest.php (hunk 1 above)
Comment (ruudboon) [reply tests/unit/Security/Random/Base64SafeCest.php]: Ok, thnx. Wil refactor this.

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
@ruudboon Thank you!

**Pull request merged**